src/core/common/DashboardCard/index.js

```diff
@@ -41,7 +41,7 @@ const LoadingItems = ({ classes }) => {
  * @param {shape}   history
  */
 const ListBlock = ({ loading, classes, items, list, history }) => {
-    if (loading) {
+    if (!items || loading) {
```

Comment:
If the result set is empty, it shows a loader? This feels wrong to me. You should test if items is empty or undefined.

Reply:
Fixed